src/libraries/System.Runtime/tests/System.Globalization.Calendars.Tests/ISOWeek/ISOWeekTests.cs

```diff
@@ -91,29 +106,48 @@
~        }
~
~        [Theory, MemberData(nameof(GetWeekOfYear_TestData))]
~        public static void GetWeekOfYear(DateTime date, int expected)
~        {
```

Comment:
>GetWeekOfYear [](http://example.com/codeflow?start=27&length=13)

xUnit doesn't allow using two test methods with the same name. Please rename this one or even merge it the two methods. This apply to GetYear too.